src/main/java/com/onevizion/scmdb/facade/DbScriptFacade.java

```diff
@@ -159,6 +158,17 @@ public void deleteAll(Collection<SqlScript> scripts) {
                                            .collect(Collectors.toList()));
     }
 
+    public List <SqlScript> getDbScriptsWithText(List<SqlScript> scripts) {
```

Comment:
this method breaks the order of `List<SqlScript> scripts` and then rollbacks will be executed randomly

Reply:
removed this method and added Map<Long, String> readTextMapByIds(Set<Long> ids)